src/diffusers/models/attention.py

```diff
@@ -515,3 +521,268 @@ def forward(self, x, emb):
         x = F.group_norm(x, self.num_groups, eps=self.eps)
         x = x * (1 + scale) + shift
         return x
+
+
+# WIP Tune a Video classes
```

Comment:
Reminder to remove this comment when done.

Reply:
Sure ! Missed this one. Will track this